packages/core/src/ui/widgets/colorBar/enums/ColorBarScalePosition.ts

```diff
@@ -0,0 +1,4 @@
+export enum ColorBarScalePosition {
```

Comment:
require documentation

Reply:
I still don't think the name is representative. How about ColorBarRangePosition?